integration/kubernetes/confidential/sev.bats

```diff
@@ -0,0 +1,323 @@
+  echo -e "SETUP FILE - STARTED\n"
+
+  start_date=$(date +"%Y-%m-%d %H:%M:%S")
+
+  TEST_DIR="$(mktemp -d /tmp/test.XXXXXXXX)"
+
+  # Install package dependencies
+  echo "Installing required packages..."
+  esudo apt install -y docker-compose jq
+  pip install sev-snp-measure
+
+  # KBS setup and run
+  echo "Setting up simple-kbs..."
+  run_kbs
+
+  # Pull image and retrieve encryption and ssh keys
+  echo "Pulling encrypted image and setting keys..."
+  pull_encrypted_image_and_set_keys
+
+  echo "SETUP FILE - COMPLETE"
+  echo "###############################################################################"
+}
+
+setup() {
+  # Remove the service/deployment/pod if it exists
+  echo "Deleting previous test pods..."
+  esudo kubectl delete -f \
+    "${TESTS_REPO_DIR}/integration/kubernetes/confidential/fixtures/unencrypted-image-tests.yaml" 2>/dev/null || true
+  esudo kubectl delete -f \
+    "${TESTS_REPO_DIR}/integration/kubernetes/confidential/fixtures/encrypted-image-tests.yaml" 2>/dev/null || true
+
+  # Delete any previous data in the DB
+  mysql -u${KBS_DB_USER} -p${KBS_DB_PW} -h ${KBS_DB_HOST} -D ${KBS_DB} <<EOF
+    DELETE FROM secrets WHERE id = 10;
+    DELETE FROM keysets WHERE id = 10;
+    DELETE FROM policy WHERE id = 10;
+EOF
+}
+
+@test "$test_tag Test SEV unencrypted container launch success" {
+  # Add key to KBS with policy measurement
+  add_key_to_kbs_db
+  
+  # Start the pod and retrieve the name
+  esudo kubectl apply -f \
+    "${TESTS_REPO_DIR}/integration/kubernetes/confidential/fixtures/unencrypted-image-tests.yaml"
+  sleep 20
+
+  print_service_info
+  pod_ip=$(esudo kubectl get pod -o wide | grep encrypted-image-tests | awk '{print $6;}')
+
+  # Look for SEV enabled in container dmesg output
+  sev_enabled=$(ssh -i ${TEST_DIR}/encrypted-image-tests \
+    -o "StrictHostKeyChecking no" \
+    -o "PasswordAuthentication=no" \
+    -t root@${pod_ip} \
+    'dmesg | grep SEV' || true)
+
+  if [ -z "$sev_enabled" ]; then
+    >&2 echo -e "${RED}KATA CC TEST - FAIL: SEV is NOT Enabled${NC}"
+    return 1
+  else
+    echo "DMESG REPORT: $sev_enabled"
+    echo -e "${GREEN}KATA CC TEST - PASS: SEV is Enabled${NC}"
+  fi
+}
+
+@test "$test_tag Test SEV encrypted container launch failure with INVALID measurement" {
+  # Generate firmware measurement
+  local append="INVALID INPUT"
+  measurement=$(generate_firmware_measurement_with_append ${append})
+  echo "Firmware Measurement: ${measurement}"
+
+  # Add key to KBS with policy measurement
+  add_key_to_kbs_db ${measurement}
+  
+  # Start the pod and retrieve the name
+  esudo kubectl apply -f \
+    "${TESTS_REPO_DIR}/integration/kubernetes/confidential/fixtures/encrypted-image-tests.yaml"
+  sleep 20
+
+  print_service_info
+
+  # Save guest qemu kernel append to file
+  duration=$((SECONDS+20))
+  while [ $SECONDS -lt $duration ]; do
+    qemu_process=$(ps aux | grep qemu | grep append || true)
```

Comment:
Assuming that this is the only QEMU process running on the machine is error prone. Also not too long ago the stability tests were intermittently failing because of that assumption. Possible solutions:

 * Ignore any running QEMU on your query (see https://github.com/kata-containers/tests/blob/CCv0/stability/hypervisor_stability_kill_test.sh#L20 for example)
 * Obtain the QEMU pid from the sandbox ID. I think we use that approach somewhere but I glanced the files and didn't find where. Maybe inspecting the files under `/run/kata-containers/shared/sandboxes/<sandbox_id>/`

Reply:
qemu process now grepped including sandbox id. Let me know if this looks okay now.